dll/win32/comctl32/edit.c

```diff
@@ -3397,12 +3397,18 @@ static LRESULT EDIT_WM_KeyDown(EDITSTATE *es, INT key)
 static LRESULT EDIT_WM_KillFocus(HTHEME theme, EDITSTATE *es)
 {
     UINT flags = RDW_INVALIDATE | RDW_UPDATENOW;
+    HWND hwndSelf = es->hwndSelf;
 
     es->flags &= ~EF_FOCUSED;
     DestroyCaret();
     if (!(es->style & ES_NOHIDESEL))
         EDIT_InvalidateText(es, es->selection_start, es->selection_end);
     EDIT_NOTIFY_PARENT(es, EN_KILLFOCUS);
+
+    // Someone destroyed us in the EN_KILLFOCUS handler, so we can't use 'es' anymore
+    if (!IsWindow(hwndSelf))
+        return 0;
```

Comment:
This is Wine code btw.

Reply:
https://gitlab.winehq.org/wine/wine/-/merge_requests/1207

Reply:
New patch :smile: https://gitlab.winehq.org/wine/wine/-/merge_requests/1270